### ISSUE TITLE
Issue setting networking keys

### ISSUE THREAD

**Issue by loganallenc** [urbit/bridge#343]
![DEECB7DF-6641-4229-9EAE-B83541BAB55B](https://user-images.githubusercontent.com/1377557/69023965-9abb2100-0975-11ea-8077-31cabe3dc3b7.jpeg)

**Comment by loganallenc** [comment]
![88D980F8-5486-4660-8BAD-CA8A7B5232C2](https://user-images.githubusercontent.com/1377557/69024002-bc1c0d00-0975-11ea-957b-ef024900bd08.jpeg)

**Comment by Fang-** [comment]
This isn't so much a bug as it's something not covered by the docs, which for the most part assume the ideal "logged in with urbit wallet" flow.

For non-urbit-wallet cases, we don't have any way of determining their networking private keys from their login data. Setting networking keys still works, but uses non-deterministic private keys under the hood, and warns the user about this during the flow, prompting them to "download these keys now or forever lose them".

Closing as not-an-issue. I've opened urbit/docs#726 to track the "undocumented" part of this.
